### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by lucasfth** [PhilipFlyvholm/learnit-plus-plus#5] [General]
Since you are working on an extension (which I have not) I wanted to know if you are interested in some of the CSS I have created together with a friend.
This includes:
* A grid layout for the front page
* Original dark color of LearnIT (which was #242424)
* Remove the background color from the card titles

<img width="1680" alt="image" src="https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/94063597/7ebc813c-7411-45de-9a7c-587bd1656ff1">

Furthermore, I use the Arc browser so I am also able to add a link to the boost I have created. A boost is just to make it easier for other Arc users to use the custom CSS and JS someone has created.

If you are interested let me know if it is all of it or only some of it. Then I will open up a PR when I have added some of the changes

**Comment by PhilipFlyvholm** [comment]
Looks amazing! This project was also started as an Arc boost, but we ran into some limitations with sharing boosts with js and also the possibility to use packages and have a file structure.

The color changes look good, however, it contradicts the current color changes made by the extension. A possible solution would be creating a theming system, where the user can choose the themes. 

The grid part looks amazing! We have already changed the order of the sections and also added new sections so I don't know how easy it would be to add into the extension, but it would be nice to have.

I think it would be nice to have a PR with the changes if:
- The color changes could be in a separate CSS file such that it could be used in a theme system in the future
- The new sections are supported in the grid layout

**Reply by lucasfth** [reply]
Nice, I will try and look into it when I have time and of course, adhere to the things you mentioned.

**Comment by lucasfth** [comment]
<img width="1680" alt="image" src="https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/94063597/7b417331-a3a1-4928-a824-677f6382465d">
<img width="1680" alt="image" src="https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/94063597/ec80fcab-199a-46c3-9ab7-176d116ed4b6">
<img width="1680" alt="image" src="https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/94063597/77ddf76c-1149-4ee4-bc39-b4d9db33c215">
Not sure what I think of the grid yet. But it is a bit tough do do in a completely satisfying way with all of the new cards. But really nice with the additions of the new cards

**Reply by lucasfth** [reply]
Also, can I get access to push to other branches on the repo (of course not main)? Or would you like me to fork it?

**Reply by PhilipFlyvholm** [reply]
Looks like a good first version! I can see I haven't tested the new cards on light mode with those buttons. 😅
I have added you as a contributor

**Comment by lucasfth** [comment]
It will be implemented in #7
